src/main/java/uk/gov/ons/census/casesvc/client/UacQidServiceClient.java

```diff
@@ -21,21 +23,25 @@
   @Value("${uacservice.connection.port}")
   private String port;
 
-  public UacQidDTO generateUacQid(int questionnaireType) {
-
+  public List<UacQidDTO> getUacQids(Integer questionnaireType, int cacheFetch) {
     RestTemplate restTemplate = new RestTemplate();
-    UriComponents uriComponents = createUriComponents(questionnaireType);
-    ResponseEntity<UacQidDTO> responseEntity =
-        restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, null, UacQidDTO.class);
-    return responseEntity.getBody();
+
+    UriComponents uriComponents =
+        createUriComponents(questionnaireType, cacheFetch, "multiple_qids");
+    ResponseEntity<UacQidDTO[]> responseEntity =
+        restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, null, UacQidDTO[].class);
+
+    return Arrays.asList(responseEntity.getBody());
```

Comment:
Might have been easier to just do it as a `List` the whole way through, but it's a minor point